Makefile

```
@@ -54,20 +52,10 @@ include lib/Makefile
 include test/Makefile
 
 
-$(BUILD_DIR)/programs.cpio:
-	@printf "TOUCH programs.cpio\n"
-	$(SIL)touch $(BUILD_DIR)/programs.cpio
-
-
-$(PREFIX_O)/programs.o.cpio: $(PREFIX_O)programs.o $(BUILD_DIR)/programs.cpio
-	@printf "EMBED programs.cpio\n"
-	$(SIL)$(OBJCOPY) --update-section .data=$(BUILD_DIR)/programs.cpio $(PREFIX_O)programs.o --add-symbol programs=.data:0 $(PREFIX_O)programs.o.cpio
-
-
-$(PREFIX_PROG)phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf: $(OBJS) $(PREFIX_O)/programs.o.cpio
+$(PREFIX_PROG)phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf: $(OBJS)
```

```suggestion
$(PREFIX_PROG)phoenix-$(TARGET_FAMILY)-$(TARGET_SUBFAMILY).elf: $(OBJS)
```
replace `{}` with `()` also on line:
```
all: $(PREFIX_PROG_STRIPPED)phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.el
```

---

Done